native/acter/src/api/backup.rs

```diff
@@ -23,8 +23,9 @@ impl BackupManager {
         let inner = self.inner.clone();
         RUNTIME
             .spawn(async move {
+                inner.wait_for_e2ee_initialization_tasks().await;
```

Comment:
additionally, we need to make sure that all e2ee initialization tasks are done or the backup might only send some stuff leading to an incomplete backup.